lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -71,30 +78,49 @@ data CabalPackageInfo = CabalPackageInfo
 preprocessorIdentifier :: String
 preprocessorIdentifier = "__preprocessor-apply-packages"
 
-run :: MonadObelisk m => m ()
+profile
+  :: MonadObelisk m
+  => String
+  -> [String]
+  -> m ()
+profile profileBasePattern rtsFlags = withProjectRoot "." $ \root -> do
+  putLog Debug "Using profiled build of project."
+
+  time <- liftIO getCurrentTime
+  let profileBaseName = formatTime defaultTimeLocale profileBasePattern time
+
+  putLog Debug $ T.pack $ "Storing profiled data under base name of " <> profileBaseName
+
+  liftIO $ createDirectoryIfMissing False $ takeDirectory profileBaseName
+
+  outPath <- nixCmd $ NixCmd_Build $ def
```

Comment:
Hm, `nixCmd` doesn't seem to properly `setCwd (Just root)` here so this command fails in anything but the root dir.